@Suchitra-sahoo created login and sign up page using html ,css and php and mysql database to store all the registered details kindly check it out and let me know for any changes or updates.

MattipaPoojitha opened the pull request.

Suchitra-Sahoo commented:
Resolve the conflicts and mention your issue number. Once done add some screenshots too.

MattipaPoojitha commented:
issue num #11 : i  created a new branch reg and also added screen shots as specified . Kindly check whether it is conflicting now i  resolved it and added 2 to 3 branches but i guess it it conflictting with my own files in my previous branches soo, check that and kindly update me how to rectify .
